examples/image_classification.ipynb

```diff
@@ -948,7 +948,7 @@
     "normalize = Normalize(mean=feature_extractor.image_mean, std=feature_extractor.image_std)\n",
```

Comment:
<p>Could you also add updates to the processing class: feature_extractor -&gt; image_processor</p><p> </p><pre class="ql-syntax" spellcheck="false">from transformers import AutoImageProcessor

image_processor = AutoImageProcessor.from_pretrained(model_checkpoint)
image_processor
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/notebooks/pull/359/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>